packages/gatsby-theme-newrelic/README.md

```diff
@@ -419,6 +420,36 @@ All props are forwarded to the underlying `a` tag with the exception of the
 <ExternalLink href="https://newrelic.com">Link to New Relic</ExternalLink>
 ```
 
+### `Feedback`
+
+Renders feedback controls that can be used to collect user sentiment about a page. Feedback can only be submitted once per page load.
+
+```js
+import { Feedback } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Props**
+
+| Prop              | Type     | Required | Default                 | Description                                                                                                                  |
+| ----------------- | -------- | -------- | ----------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `align`           | enum     | no       | `Feedback.ALIGN.CENTER` | Configures the alignment of the feedback component. Must be one of `Feedback.ALIGNMENT.LEFT` or `Feedback.ALIGNMENT.CENTER`. |
+| `onPositiveClick` | function | yes      |                         | Callback function that is called once the positive button is clicked.                                                        |
+| `onNegativeClick` | function | yes      |                         | Callback function that is called once the negative button is clicked.                                                        |
```

Comment:
```suggestion
| `onNegativeClick` | function | yes      |                         | Handler that is called once the negative button is clicked.                                                        |
```

The rest of the README uses the term `handler` to refer to these callback functions. This will put it more in alignment with the rest of the components.